datagateway_api/src/datagateway_api/database/models.py

```diff
@@ -567,6 +851,25 @@ class FACILITYCYCLE(Base, EntityHelper, metaclass=EntityMeta):
     )
 
 
+class FUNDINGREFERENCE(Base, EntityHelper, metaclass=EntityMeta):
+    __tablename__ = "FUNDINGREFERENCE"
+    __singularfieldname__ = "fundingReference"
+    __pluralfieldname__ = "datasetTechniques"
```

Comment:
I think this is incorrect, could you correct it please? `fundingReferences` is the correct plural field name

Reply:
Yes. I have corrected that.